refactor: update `math/base/assert/int32-is-even` to follow latest project conventions

vivekmaurya001 opened the pull request:
Resolves None.

## Description

- refactoring addon.c to follow latest project conventions for  `math/base/assert/int32-is-even` 
- updating benchmark calculations

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

 - [x] Read, understood, and followed the [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md)


* * *

@stdlib-js/reviewers 

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

vivekmaurya001 committed:
refactor: update math/base/assert/int32-is-even to follow latest project conventions

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: na
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: na
  - task: run_c_benchmarks
    status: passed
  - task: run_cpp_benchmarks
    status: na
  - task: run_fortran_benchmarks
    status: na
  - task: run_javascript_benchmarks
    status: passed
  - task: run_julia_benchmarks
    status: na
  - task: run_python_benchmarks
    status: na
  - task: run_r_benchmarks
    status: na
  - task: run_javascript_tests
    status: na
---

vivekmaurya001 committed:
Merge branch 'int32-is-even' of https://github.com/vivekmaurya001/stdlib into int32-is-even

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: na
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: na
  - task: run_c_benchmarks
    status: passed
  - task: run_cpp_benchmarks
    status: na
  - task: run_fortran_benchmarks
    status: na
  - task: run_javascript_benchmarks
    status: passed
  - task: run_julia_benchmarks
    status: na
  - task: run_python_benchmarks
    status: na
  - task: run_r_benchmarks
    status: na
  - task: run_javascript_tests
    status: na
---

vivekmaurya001 commented:
good afternoon , @kgryte please give this a reveiw

vivekmaurya001 commented on lib/node_modules/@stdlib/math/base/assert/int32-is-even/benchmark/benchmark.js:
```diff
@@ -31,14 +31,17 @@ var isEven = require( './../lib' );
+
 	b.tic();
 	for ( i = 0; i < b.iterations; i++ ) {
-		x = ( randu()*1.0e7 ) - 5.0e6;
```
the number that's get generated here is not getting round , is it a mistake or intentional ?

kgryte replied:
Likely a mistake. Feel free to fix.

vivekmaurya001 replied:
ooh sorry i realised that i have been using a extra package float64toInt32 rather than using opts in discreteuniform

vivekmaurya001 replied:
i have corrected it

vivekmaurya001 committed:
refactor: update math/base/assert/int32-is-even to follow latest project conventions

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: na
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: na
  - task: run_c_benchmarks
    status: na
  - task: run_cpp_benchmarks
    status: na
  - task: run_fortran_benchmarks
    status: na
  - task: run_javascript_benchmarks
    status: passed
  - task: run_julia_benchmarks
    status: na
  - task: run_python_benchmarks
    status: na
  - task: run_r_benchmarks
    status: na
  - task: run_javascript_tests
    status: na
---

vivekmaurya001 committed:
Merge branch 'stdlib-js:develop' into int32-is-even

vivekmaurya001 committed:
Merge branch 'stdlib-js:develop' into int32-is-even

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
---

vivekmaurya001 committed:
refactor: update math/base/assert/int32-is-even to follow latest project conventions

vivekmaurya001 committed:
Merge branch 'int32-is-even' of https://github.com/vivekmaurya001/stdlib into int32-is-even

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: na
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: na
  - task: run_c_benchmarks
    status: na
  - task: run_cpp_benchmarks
    status: na
  - task: run_fortran_benchmarks
    status: na
  - task: run_javascript_benchmarks
    status: na
  - task: run_julia_benchmarks
    status: na
  - task: run_python_benchmarks
    status: na
  - task: run_r_benchmarks
    status: na
  - task: run_javascript_tests
    status: na
---

vivekmaurya001 commented:
im not able to get where error is occuring
